rust/azure_iot_operations_connector/Cargo.toml

```
@@ -17,8 +17,12 @@ publish = true
 # azure_iot_operations_mqtt = { version = "0.9", path = "../azure_iot_operations_mqtt", registry = "aio-sdks"  }
 # derive_builder.workspace = true
 # log.workspace = true
-# tokio.workspace = true
+tokio.workspace = true
 # tokio-util.workspace = true
+notify = "5.1"
+futures = "0.3"
+tokio-stream = "0.1"
+thiserror = "1.0"
```

thiserror you can use from the workspace too - although we're using version 2.0.12, any reason you're using 1.0?
```suggestion
thiserror.workspace = true
```